packages/plugin-ext/src/hosted/node/scanners/backend-init-theia.ts

```diff
@@ -29,7 +30,7 @@ export const doInitialization: BackendInitializationFn = (apiFactory: PluginAPIF
     const apiImpl = apiFactory(plugin);
     pluginsApiImpl.set(plugin.model.id, apiImpl);
 
-    plugins.push(plugin);
+    plugins.push({ ...plugin, pluginFolder: realpathSync(plugin.pluginFolder) });
```

Comment:
Suggestion: I don't believe this is necessary anymore, given that we already store the resolved plugin path in the `plugin.pluginFolder`.